### PR TITLE
[INLONG-810][Doc] Update version release related docs

### LINKED ISSUE #810: [Improve] Update version release related docs
### Description

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #810 

### Motivation

Update version release related docs

### CODE REVIEW

File: community/how-to-release.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ Uploading takes about one minute, after that, you can check it by your email at
 The svn repository of the DEV branch is: https://dist.apache.org/repos/dist/dev/inlong
 
 The svn repository of the Release branch is: https://dist.apache.org/repos/dist/release/inlong
+
+Please ensure that the key you add to dev and release branch are the same.
```
Comment (dockerzhang) [inline community/how-to-release.md:122]: ->
Please ensure the `key` you added to the `dev` and `release` branches is the same.

File: community/how-to-release.md [hunk 2]
```diff
@@ -287,7 +289,8 @@ $ svn commit -m "prepare for ${release_version} ${rc_version}"
 - You will get a link like `https://repository.apache.org/content/repositories/orgapacheinlong-{staging-id}` on the Summary page if all checks passed. This link will be posted in the vote email, so, keep it safe.
 
 :::caution
-Close operation may fail, you should check the causes and fix them.
+- Close operation may fail, you should check the causes and fix them.
+- The key you upload before may not be synchronized among keyserver, please upload it to keyserver.ubuntu.com or keys.openpgp.org if staging fail by error ![img.png](images/staging-error.png)
```
Comment (dockerzhang) [inline community/how-to-release.md:293]: ——>
- The close operation may not work; it's better to check and fix the causes.
- The key you uploaded before may not be synchronized among the keyserver. Please upload it to `keyserver.ubuntu.com` or `keys.openpgp.org` if there is a `Signature Validation` event.

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md [hunk 3]
```diff
@@ -116,6 +116,8 @@ $ gpg --keyserver pgpkeys.mit.edu --send-key <key id>
 dev 分支的 svn 仓库：https://dist.apache.org/repos/dist/dev/inlong
 
 release 分支的 SVN 仓库：https://dist.apache.org/repos/dist/release/inlong
+
+请确认dev和release中增加的公钥相同
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md:120]: 请确认 `dev` 和 `release` 分支中增加的 `key` 相同

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md [hunk 4]
```diff
@@ -285,7 +287,8 @@ $ svn commit -m "prepare for ${release_version} ${rc_version}"
 - 检查通过以后, 在下方的 Summary 标签页上出现一个连接，请保存好这个链接，需要放在接下来的投票邮件当中，链接类似 `https://repository.apache.org/content/repositories/orgapacheinlong-{staging-id}` 。
 
 :::caution
-请注意点击 Close 可能会出现失败，请检查失败原因并处理
+- 请注意点击 Close 可能会出现失败，请检查失败原因并处理
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md:290]: - 如果 Close 操作失败，请检查失败原因并处理
- 如果出现 `Signature Validation` 错误事件，可能是 keyserver 之间没有同步 key，可以重新上传 key 到 `keyserver.ubuntu.com`  或 `keys.openpgp.org`。

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md [hunk 5]
```diff
@@ -116,6 +116,8 @@ $ gpg --keyserver pgpkeys.mit.edu --send-key <key id>
 dev 分支的 svn 仓库：https://dist.apache.org/repos/dist/dev/inlong
 
 release 分支的 SVN 仓库：https://dist.apache.org/repos/dist/release/inlong
+
+请确认`dev`和`release`分支中增加的`key`相同
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md:120]:
```suggestion
请确认 `dev` 和 `release` 分支中增加的 `key` 相同
```

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md [hunk 6]
```diff
@@ -285,7 +287,8 @@ $ svn commit -m "prepare for ${release_version} ${rc_version}"
 - 检查通过以后, 在下方的 Summary 标签页上出现一个连接，请保存好这个链接，需要放在接下来的投票邮件当中，链接类似 `https://repository.apache.org/content/repositories/orgapacheinlong-{staging-id}` 。
 
 :::caution
-请注意点击 Close 可能会出现失败，请检查失败原因并处理
+- 如果Close操作失败，请检查失败原因并处理
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md:290]:
```suggestion
- 如果 Close 操作失败，请检查失败原因并处理
```

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md [hunk 7]
```diff
@@ -285,7 +287,8 @@ $ svn commit -m "prepare for ${release_version} ${rc_version}"
...
+- 如果出现 `Signature Validation` 错误事件，可能是 keyserver 之间没有同步 key，可以重新上传 key 到 `keyserver.ubuntu.com`  或 `keys.openpgp.org`
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-release.md:291]:
```suggestion
- 如果出现 `Signature Validation` 错误事件，可能是 `keyserver` 之间没有同步 `key`，可以重新上传 `key` 到 `keyserver.ubuntu.com`  或  `keys.openpgp.org`
```

File: i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-verify.md [hunk 8]
```diff
@@ -53,7 +53,7 @@ for i in *.tar.gz; do echo $i; gpg --verify $i.asc $i ; done
 :::
 
 ### 检查二进制包
-解压缩 `apache-inlong-${release_version}-bin.tar.gz` 和 `apache-inlong-${release_version}-sort-connectors.tar.gz`，进行如下检查:
+解压缩 `apache-inlong-${release_version}-bin.tar.gz`, 以及对应Flink版本的`Sort-Connector`，进行如下检查:
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs-community/current/how-to-verify.md:56]:
```suggestion
解压缩 `apache-inlong-${release_version}-bin.tar.gz`, 以及对应 Flink 版本的 `Sort Connector`，进行如下检查:
```

File: community/how-to-verify.md [hunk 9]
```diff
@@ -53,7 +53,7 @@ You can check the compilation and unit test through `mvn clean package install`.
 :::
 
 ### Check binary packages
-Unzip `apache-inlong-${release_version}-bin.tar.gz` and `apache-inlong-${release_version}-sort-connectors.tar.gz` and check the following:
+Unzip `apache-inlong-${release_version}-bin.tar.gz`, and the `Sort-Connector` corresponding to your Flink version, then check the following:
```
Comment (dockerzhang) [inline community/how-to-verify.md:56]:
```suggestion
Unzip `apache-inlong-${release_version}-bin.tar.gz`, and the `Sort Connector` corresponding to your Flink version, then check the following:
```

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**